### PR TITLE
Fixes a bug where the Rollbar integration was loading twice

### PR DESCRIPTION
**Pull request by coryvirok**
The integration was previously loading the full Rollbar javascript code asynchronously in the `initialize()` method and again in the `load()` method. This was done in order to ensure the Rollbar interface was available once the callback passed to `load()` was executed. This was unnecessary.

This change loads the full Rollbar javascript only once, inside the `initialize()` method. The `load()` method has been modified to wait for the full source to download before it calls the `load()` callback.

### CODE REVIEW

**Comment by hankim813** [comment]
@coryvirok thanks so much for the PR.

lgtm but going to get more eyes on this before we merge and deploy. 

@f2prateek @sperand-io can you guys look over this as well?

File: test/index.html [hunk 1]
```diff
@@ -2,12 +2,12 @@
 <html>
   <head>
     <title>integrations tests</title>
-    <link rel="stylesheet" href="/node_modules/mocha/mocha.css">
+    <link rel="stylesheet" href="../node_modules/mocha/mocha.css">
```
Comment (f2prateek) [inline test/index.html:5]: Why did this change?

**Comment by f2prateek** [comment]
lgtm, can you squash and rebase these changes?

File: test/index.test.js [hunk 2]
```diff
@@ -44,13 +47,11 @@ describe('Rollbar', function() {
       it('should create the window.Rollbar object', function() {
         analytics.assert(!window.Rollbar);
         analytics.initialize();
-        analytics.page();
         analytics.assert(window.Rollbar);
       });
 
       it('should have all of the correct methods', function() {
         analytics.initialize();
-        analytics.page();
```
Comment (f2prateek) [inline test/index.test.js:53]: Why are these removed?

**Comment by hankim813** [comment]
hey @coryvirok, if you could clarify some of our comments we can push this pr along. Thanks!

File: test/index.html (hunk 1 above)
Comment (coryvirok) [inline test/index.html:5]: I changed this so that I could use my IDE, (PyCharm) to host the HTML files and test, (instead of Express.)

I've verified that `make test-browser` still works correctly.

File: test/index.test.js (hunk 2 above)
Comment (coryvirok) [inline test/index.test.js:53]: These weren't actually doing anything. I think I originally copied them from some other integration's tests but didn't check to see if they were required.

E.g. We don't need `page()` to be called in order to verify that the Rollbar notifier has all of the appropriate methods after initialization.

**Comment by coryvirok** [comment]
I don't think my attempt at squashing the commits worked like I expected... I'm happy to tidy up the commits, just let me know how.

**Comment by hankim813** [comment]
Hey @coryvirok,

This is basically what you want to do for the rebasing:

I would just revert back to your original commits first. If it's messy, then just make a new PR :)

Before you push to the remote branch:

```
// when you're ready to commit
git add .
git commit -m 'update rollbar init'
git checkout master
git fetch origin master
git rebase origin/master
git checkout <branch>
git rebase -i master
git push origin <branch>
```

Thanks!

**Comment by hankim813** [comment]
@coryvirok any updates on this?

**Comment by hankim813** [comment]
@coryvirok hey cory -- let me know if you can do this by end of today. if not I'm going to just make a new PR with the same changes and push that up since the mutual client has been waiting for a while now. 

Thanks!

**Comment by coryvirok** [comment]
Sorry for the delayed response, @hankim813. I didn't see this until just now. Please make the new PR. Thx

**Comment by coryvirok** [comment]
Followed your instructions and made a new PR here: https://github.com/segment-integrations/analytics.js-integration-rollbar/pull/8
